pkg/postgres/postgres_test.go

```diff
@@ -80,7 +90,7 @@ func TestConnectionString_String(t *testing.T) {
 		Host:     "host:5432",
 		Database: "database",
 		User:     "user",
-		Password: "1234",
+		Password: "12:34",
```

Comment:
How come this was needed?

Reply:
I wanted a test to fail if the password contained special characters per the fix in postgres.go, but I'll move both of these changes out into a separate PR per your request.